### PR TITLE
Search bar is aligned

### LINKED ISSUE #508: Search bar must be present on a single line
### What happened just before the problem occurred
The search bar is not present on a single line.
Selection_126

It must be present in a single line. 

### Relevant URLs
https://mapknitter.org/

### PublicLab.org username
divyabaid16

(to help reproduce the issue)

### Browser, version, and operating system
Linux Mint

For bug reports, fill out the above template; for feature requests, you can delete the template.

### PR DESCRIPTION
**Pull request by divyabaid16**
Fixes #508  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

GIF for the change:
Selection_127

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

Thanks!

### CODE REVIEW

File: app/views/maps/_search.html.erb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <h5>Looking for something on MapKnitter?</h5>
 <form action="/search" method='get' class="navbar-form navbar-left">
   <div class="input-group">
-    <input class="form-control" id="search-input" style="min-width: 280px;" type="text" name='q' placeholder="Search maps" value='<%= params[:id] if params[:action] == "search" %>'>
-    <span class="input-group-append">
+    <input style="display: inline-block;" class="form-control" id="search-input" style="min-width: 280px;" type="text" name='q' placeholder="Search maps" value='<%= params[:id] if params[:action] == "search" %>'>
```
Comment (gauravano) [inline app/views/maps/_search.html.erb]:
```suggestion
    <input class="form-control" id="search-input" style="min-width: 280px; display: inline-block;" type="text" name='q' placeholder="Search maps" value='<%= params[:id] if params[:action] == "search" %>'>
```

Comment (gauravano) [reply app/views/maps/_search.html.erb]: @divyabaid16 how about using the already present style attribute?

**Comment by divyabaid16** [comment]
@gauravano I have made the changes.
Thanks!

**Comment by sashadev-sky** [comment]
@divyabaid16 @gauravano the other implementation didn't fix the issue for me and I looked into it and found that the main issue was that we put the `min-width` style attribute on the input instead of the whole div (input + the search icon box). 

I moved it up to the `div` and it should be fixed now! @divyabaid16 let me know if this fixed it for you. I just pushed the commit here because it was easier but I will revert it if necessary

**Comment by sashadev-sky** [comment]
@publiclab/reviewers

**Comment by sashadev-sky** [comment]
closing this because this search bar has been deleted... will open a PR to fix the other one!

**Pull request closed without merging**